.circleci/config.yml

```diff
@@ -1308,6 +1342,8 @@ workflows:
           test_list: git diff origin/develop...HEAD --name-only -- './test/**/*.t.sol' | sed 's|packages/contracts-bedrock/||'
           test_timeout: 1h
           test_profile: ciheavy
+      - diff-asterisc-bytecode:
+          name: diff-asterisc-bytecode
```

Comment:
Won't this fail as soon as upstream asterisc changes and then create errors and block all PRs merging?  Seems like this would be a better fit as a scheduled job rather than running on every PR.

Reply:
Scheduled job sounds good to me. Who should it tag in slack on failure, @clabby and @refcell?

Reply:
tagged myself and @proofs-squad. I'm currently on @proofs-squad in slack, @'d myself for redundancy against getting ousted 😢